airflow/api_fastapi/execution_api/datamodels/taskinstance.py

```diff
@@ -137,5 +137,12 @@ class TaskInstance(BaseModel):
     map_index: int | None = None
 
 
+"""Defines the types that the RTIF payload's dictionary values can take. These are all JsonAble types """
+JsonAbleValueTypes = Union[
+    dict[str, JsonValue],
+    list[JsonValue],
+    JsonValue,
+]
+
```

Comment:
Actually now you can just remove this entirely and change L148 to use `JsonValue`

https://docs.pydantic.dev/2.9/api/types/#pydantic.types.JsonValue

It is sufficient for what we want:

```
JsonValue: TypeAlias = Union[
    List["JsonValue"],
    Dict[str, "JsonValue"],
    str,
    bool,
    int,
    float,
    None,
]
```